msteams-platform/tabs/design/tabs.md

```diff
@@ -138,6 +138,8 @@ Notify users of tab activity one of the following ways:
 
 ## Best practices
 
+Following these best practices can help ensure a quality user experience. If want to list your app in the Teams store, make sure your design also adheres the [store validation guidelines](~/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md).
```

Comment:
Should we include this disclaimer in our design guidelines best practices so users better understand what these are compared to similar validation guidelines?